include/shapes/polygon.hpp

```diff
@@ -0,0 +1,19 @@
+#ifndef _RIVE_POLYGON_HPP_
+#define _RIVE_POLYGON_HPP_
+#include "generated/shapes/polygon_base.hpp"
+#include <stdio.h>
+namespace rive
+{
+	class Polygon : public PolygonBase
```

Comment:
Make sure you add a ~Polygon() destructor to clear vertices or the ones you new will remain on the heap forever-ever.

<img src="https://media.tenor.com/images/9ea65d4da91faee1f810acb30bf9557d/tenor.gif"/>

Reply:
Added